add-hotel.php

```diff
@@ -0,0 +1,128 @@
+                                <input type="email" class="w-100" id="h-email">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Contact</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="h-contact">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>No. of Rooms</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="h-rooms">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Rooms Numbers</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <div class="row">
+                                    <div class="col-12">
+                                        <input type="text" class="w-100" class="w-100" id="room-numbers" placeholder="Use ',' to seperate room numbers">
+                                    </div>
+                                    <div class="col-12">
+                                        <div class="row">
+                                            <div class="col-6">
+                                                <input type="radio" name="type" value="a/c"> A/C
+                                            </div>
+                                            <div class="col-6">
+                                                <input type="radio" name="type" value="n-a/c"> Non A/C
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Price Per. Room (LKR)</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="h-price">
```

Comment:
If you have to use an input element for `price`. You can use an attribute called `pattern`. It helps to prevent invalid value entered by the user. 

Refer [this](https://stackoverflow.com/questions/29603080/html5-input-type-number-with-regex-pattern)

Reply:
Got it